samples/boards/nordic/coresight_stm/README.rst

```diff
@@ -0,0 +1,102 @@
+.. zephyr:code-sample:: coresight_stm_sample
+   :name: Coresight STM benchmark
+   :relevant-api: log_api
+
+Overview
+********
+
+This sample presents how to enable STM logging on nRF54H20 platform.
+
+Also, it prints timing for different log messages.
+Thus, performance of different loggers can be compared.
+
+There are three sample configurations in the :file:`sample.yaml`.
+
+* **sample.boards.nrf.coresight_stm.local_uart**
+
+This configuration doesn't use STM.
+Logs are printed on local console.
+
+* **sample.boards.nrf.coresight_stm**
+
+This configuration use STM.
+Application, Radio, PPR and FLPR cores send logs to an ETR buffer.
+Proxy (Application core) gets logs from the ETR buffer, decodes STPv2 data
+and prints human readable logs on UART.
+
+* **sample.boards.nrf.coresight_stm.dict**
+
+This sample uses STM logging in dictionary mode.
+Application, Radio, PPR and FLPR cores send logs to the ETR buffer.
+Proxy (Application core) forwards data from the ETR to the host using UART.
+Host tool is needed to decode the logs.
```

Comment:
```suggestion
**sample.boards.nrf.coresight_stm.local_uart**
  This configuration doesn't use STM.
  Logs are printed on local console.

**sample.boards.nrf.coresight_stm**
  This configuration use STM.
  Application, Radio, PPR and FLPR cores send logs to an ETR buffer.
  Proxy (Application core) gets logs from the ETR buffer, decodes STPv2 data
  and prints human readable logs on UART.

**sample.boards.nrf.coresight_stm.dict**
  This sample uses STM logging in dictionary mode.
  Application, Radio, PPR and FLPR cores send logs to the ETR buffer.
  Proxy (Application core) forwards data from the ETR to the host using UART.
  Host tool is needed to decode the logs.
```

Reply:
Yes, I don't know how to move 'sample configuration' description to next line.

Generated output:
* **sample.boards.nrf.coresight_stm.local_uart** This configuration doesn’t use STM. Logs are printed on local console. 

I wanted following:
* **sample.boards.nrf.coresight_stm.local_uart**
  This configuration doesn’t use STM. Logs are printed on local console. 

I will revert to the 'definition list'.

Reply:
Mmmh no what you have now is a regular bullet list (starts with `*`), so regarding "I don't know how to move 'sample configuration' description to next line." it's because you need empty lines to actually trigger paragraph breaks.

So either

```rst
* **a bullet list**

  with an indented block that's
  flush with the beginning of the list item, and

  blank lines whenever you want a

  line return (just like any other rst text block, actually) 

```

or

```rst
** a definition list**
     with an indented block for
     the contents

     and there too, blank lines may be used

     when you want line returns in the rendered text
```

Reply:
Thank You!